packages/toolkit/src/api.ts

```diff
@@ -4,12 +4,36 @@
  */
 
 import { Auth, Connection } from 'aws-core-vscode/auth'
+
+export interface AwsToolkitSsoConnection {
+    readonly id: string
+    readonly label: string
+    readonly type: string
+    readonly ssoRegion: string
+    readonly startUrl: string
+    readonly scopes?: string[]
+}
+
 /**
  * Exposing listConnections API for other extension to read or re-use
  * the available connections in aws toolkit.
  */
 export const awsToolkitApi = {
-    async listConnections(): Promise<Connection[]> {
-        return Auth.instance.listConnections()
+    async listSsoConnections(): Promise<AwsToolkitSsoConnection[]> {
```

Comment:
- `listConnections` can return all kinds of connections. It might only return SSO for now, but we should leave the door open for other kind sin the future.
- we can name our public interfaces `AwsXX`, we don't need to repeat "Toolkit" everywhere. It will be in a namespaced module.

```suggestion
    async listSsoConnections(): Promise<AwsConnection[]> {
```

Reply:
Fixed in latest commit. Thanks.